server/blindai_sgx/src/model_store.rs

```diff
@@ -36,12 +43,35 @@ impl ModelStore {
 
     pub fn add_model(
         &self,
+        model_path: &Path,
         model_bytes: &[u8],
         input_facts: Vec<Vec<usize>>,
         model_name: Option<String>,
+        model_id: Uuid,
         datum_inputs: Vec<ModelDatumType>,
         datum_outputs: Vec<ModelDatumType>,
     ) -> Result<(Uuid, Digest)> {
+
+        // Sealing/////////////////////////////////////////////////////
+        sealing::seal(
```

Comment:
what if the user supplied `sealing = false` in the request, which means they don't want the model to be sealed?